.github/actions/report-ci-errors/README.md

```diff
@@ -0,0 +1,26 @@
+## Pre-requisites
+
+You will need to get the slack webhook url from the secrets manager first and add it as SLACK_WEBHOOK_URL as a repository secret. 
```

Comment:
We want to move away from ClickOpsing GitHub config so ideally it should be added in the Terraform config. Similar to organisations secrets https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/main.tf#L224-L253

As per our chat on Slack: https://gds.slack.com/archives/C079ULB1UEP/p1740663568473939